docs/networking/networking.md

```diff
@@ -79,17 +79,19 @@ To select a particular subset of nodes to host pods for a LoadBalancer, set matc
 
 ### Disabling the Service LB
 
-To disable the embedded LB, configure all servers in the cluster with the `--disable=servicelb` option.
+To disable the embedded LB, configure all servers in the cluster with the `--disable=servicelb` flag.
 
 This is necessary if you wish to run a different LB, such as MetalLB.
 
 ## External Cloud Controller Manager
 
-K3s ships with a stub Cloud Controller Manager (CCM) that does not actually implement any functionality. In order to use an external CCM, you must start K3s with the `--disable-cloud-controller` flag. 
+K3s does not ship with any "in-tree" cloud providers. Instead, K3s ships with a Cloud Controller Manager (CCM) stub that does the following:
```

Comment:
```suggestion
In order to reduce binary size, K3s removes all "in-tree" (built-in) cloud providers. Instead, K3s provides an embedded Cloud Controller Manager (CCM) that does the following:
```

Reply:
👍🏻 / 👎🏻 ?

Reply:
I added this in https://github.com/k3s-io/docs/pull/98/commits/8327f649a9dbed67a7f7fd0931f83ef56479381a